packages/x-charts/src/models/axis.ts

```diff
@@ -427,6 +427,34 @@ export type AxisValueFormatterContext<S extends ScaleName = ScaleName> =
       tickNumber?: number;
     };
 
+type MinMaxConfig<S extends ScaleName = ScaleName> = S extends ContinuousScaleName
+  ? S extends 'utc' | 'time'
+    ? {
+        /**
+         * The minimal value of the domain.
+         * If not provided, it gets computed to display the entire chart data.
+         */
+        min?: Date;
```

Comment:
We can probably just use `NumberValue` instead. That way date instances from other libraries should be interchangeable (as long as they implement `valueOf()`).